ui/app/views/deployments/new.html

```diff
@@ -184,6 +184,9 @@ <h1 class="page-header">New Deployment</h1>
                             <button class="btn btn-outline btn-primary pull-right" data-toggle="modal" data-target="#latest-commit-from-branch">
                                 Deploy latest commit from branch
                             </button>
+                            <button class="btn btn-outline btn-primary pull-right" data-toggle="modal" style="margin-right:10px" data-target="#find-my-commit">
```

Comment:
I'll look the other way if you fix this really really fast.
No inline style, never. There are classes for everything, use them